client/src/components/ObjectEditor.vue

```diff
@@ -17,7 +17,7 @@
                         v-if="prop.propType == 'editable'"
                         @click="handleEditablePropClick($event, prop)"
                         @blur="handleEditablePropBlur($event)"
-                        @keydown="handleEditablePropKeyDown($event, prop)"
+                        @keydown="handleEditablePropKeyDown($event, prop, objectEditorData)"
```

Comment:
You don't have to pass in `objectEditorData` here. The entire Vue component already has access to it via `this. objectEditorData` within the functions of the Vue component.